.github/renovate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>oxidecomputer/renovate-config",
+    "local>oxidecomputer/renovate-config//rust/autocreate",
+    "local>oxidecomputer/renovate-config:post-upgrade",
```

Comment:
By using this "extends" statement, we automatically parse https://github.com/oxidecomputer/renovate-config/pull/5 's `post-upgrade.json` and run the associated script? Or is there some other registration that's happening here?

Reply:
Yes -- it just fetches that JSON file and includes that in the config.